sea.js

```diff
@@ -217,6 +227,13 @@
       if(d){ jwk.d = d }
       return jwk;
     };
+    
+    s.keyTojwk = function(keyBytes) {
+      var jwkKeyString = keyBytes.toString('base64')
+      jwkKeyString = jwkKeyString.replace(/\+/g, "-").replace(/\//g, "_").replace(/\=/g, "");
+      jwkKeyString = { kty: "oct", k: jwkKeyString, ext: false, alg: "A256GCM"};
+      return jwkKeyString
+    }
```

Comment:
Maybe `s.keyTojwk` is better to rename full camelCase `s.keyToJwk` refactor like this:
```
    s.keyToJwk = function(keyBytes) {
      const keyB64 = keyBytes.toString('base64');
      const k = keyB64.replace(/\+/g, "-").replace(/\//g, "_").replace(/\=/g, "");
      return { kty: 'oct', k: k, ext: false, alg: 'A256GCM' };
    }
```

Reply:
done